cmd/influxd/inspect/inspect.go

```diff
@@ -2,10 +2,11 @@ package inspect
 
 import (
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
 // NewCommand creates the new command.
-func NewCommand() *cobra.Command {
+func NewCommand(_ *viper.Viper) *cobra.Command {
```

Comment:
Feels inconsistent to me to require the viper arg here (but not use it), and not do the same for `NewExportIndexCommand()`. Is there a reason why it needs to be passed here?

Reply:
I'll remove it for now – originally I wanted the consistency at the call site for all the `NewCommand` calls, and given there are a large number of subcommands to be added here, it was possible one might require it. We can leave that for another day.

Reply:
Done – removed the unused argument